src/util/HeaderUtil.ts

```diff
@@ -414,6 +415,17 @@ export function addHeader(response: HttpResponse, name: string, value: string |
   response.setHeader(name, allValues.length === 1 ? allValues[0] : allValues);
 }
 
+/**
+ * Parses out the first value of an X-Forwarded-* header.
+ *
+ * @param value - The value of the X-Fowarded-* header.
+ *
+ * @returns The first value of the X-Fowraded-* header.
+ */
+function parseXForwarded(value = ''): string {
+  return value.split(',').map((val): string => val.trim())[0];
```

Comment:
```suggestion
  return value.trim().replace(/\s*,.*/, '');
```

Reply:
And given its simplicity now, let’s inline it?

Reply:
Pulled this in and inlined it.